example/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "region" {
+  default = "us-east-1"
+}
+
+variable "attributes" {
+  default = []
+}
+
+variable "namespace" {
+  default = "eg"
+}
+
+variable "name" {
+  default = "efs"
+}
+
+variable "stage" {
+  default = "testing"
+}
+
+variable "availability_zones" {}
```

Comment:
Please update to this format to make it consistent:
variable "availability_zones" {
   type = "list"
}

Add `type = "list"` for all list types .